arena_bringup/launch/testing/move_base/move_base_arena.launch

```diff
@@ -1,19 +1,27 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <launch>
     <!-- <arg name="speed" default="0.22"/> -->
-    <arg name="model" default="turtlebo3_burger"/>
+    <arg name="model" default="turtlebo3_burger" />
```

Comment:
turtlebot3_burger is not a name of one of our robots, we should use burger here

Reply:
Right! Changed it to burger.